src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_northrend_beasts.cpp

```diff
@@ -388,6 +382,7 @@ class npc_snobold_vassal : public CreatureScript
                     return;
 
                 ScriptedAI::AttackStart(who);
+                SetCombatMovement(false);
```

Comment:
SetCombatMovement only needs to be called once (it just flips a bool flag on ScriptedAI). ctor is fine.